backend/src/main/java/se/kth/iv1201/recruitmentapplicationgroup5/controller/AccountController.java

```diff
@@ -0,0 +1,34 @@
+package se.kth.iv1201.recruitmentapplicationgroup5.controller;
+
+import java.net.URI;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import se.kth.iv1201.recruitmentapplicationgroup5.model.dto.AccountDTO;
+import se.kth.iv1201.recruitmentapplicationgroup5.model.dto.RegistrationDetails;
+import se.kth.iv1201.recruitmentapplicationgroup5.service.AccountService;
+
+@RestController
+@Validated
+public class AccountController {
+
+	@Autowired
+	private AccountService service;
+
```

Comment:
Field dependency injection not recommended for required dependencies, use constructor injection instead.